src/client-side-encryption/state_machine.ts

```diff
@@ -543,15 +576,19 @@ export class StateMachine {
   fetchKeys(
     client: MongoClient,
     keyVaultNamespace: string,
-    filter: Uint8Array
+    filter: Uint8Array,
+    timeoutContext?: TimeoutContext
   ): Promise<Array<DataKey>> {
     const { db: dbName, collection: collectionName } =
       MongoDBCollectionNamespace.fromString(keyVaultNamespace);
 
     return client
       .db(dbName)
       .collection<DataKey>(collectionName, { readConcern: { level: 'majority' } })
-      .find(deserialize(filter))
+      .find(deserialize(filter), {
```

Comment:
Lots of red in CI caused by this - `timeoutMode` can only be specified if timeoutMS is specified too.  Otherwise cursors throw an error.

Also - lint is failing

Reply:
Yep, changed it!